src/components/BooksForm.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
- [ ] Similar to `BookList` component, `BooksForm` component should be in `containers` folder since it is going to connect to Redux store to add a book in the following milestones.

Reply:
BooksForm.js has now been relocated to the containers folder.